sessions-core/src/main/java/com/adyen/checkout/sessions/model/setup/SessionSetupResponse.kt

```diff
@@ -66,7 +68,11 @@ data class SessionSetupResponse(
                             jsonObject.optJSONObject(PAYMENT_METHODS),
                             PaymentMethodsApiResponse.SERIALIZER
                         ),
-                        returnUrl = jsonObject.optString(RETURN_URL)
+                        returnUrl = jsonObject.optString(RETURN_URL),
+                        configuration = ModelUtils.deserializeOpt(
```

Comment:
You forgot to add this to the `serialize` function as well

Reply:
Do we use the `serialize` method for responses? I mean it's nice to have both, but do we actually use it?

Reply:
We don't use it but for the sake of consistency let's do it, since we serialize the rest of the fields